.github/workflows/build_documentation.yml

```diff
@@ -76,41 +77,24 @@ jobs:
           git config --global user.name "Hugging Face Doc Builder"
           git config --global user.email docs@huggingface.co
           
-          cd doc-build
-          git pull origin main
-          cd ..
-
-          cd notebooks
-          git pull origin master
-          cd ..
-
       - name: Make documentation
         run: |
           cd doc-builder &&
-          doc-builder build transformers ../transformers/docs/source --build_dir ../doc-build --notebook_dir notebooks/transformers_doc --clean --html &&
+          doc-builder build transformers ../transformers/docs/source --build_dir ../build_dir --notebook_dir ../notebooks_dir --clean --html &&
           cd ..
         env:
           NODE_OPTIONS: --max-old-space-size=6656
 
       - name: Push to repositories
         run: |
           cd doc-build &&
-          if [[ `git status --porcelain` ]]; then 
-            git add . &&
-            git stash && git pull && git stash apply &&
-            git commit -m "Updated with commit ${{ github.sha }} \n\nSee: https://github.com/huggingface/transformers/commit/${{ github.sha }}" &&
-            git push origin main
-          else
-            echo "No diff in the documentation."
-          fi &&
+          git pull &&
+          mv ../builder_dir/transformers . &&
```

Comment:
```suggestion
          rm -rf ./transformers/master
          mv ../builder_dir/transformers ./transformers &&
```

Reply:
Tried with `cp -r` but will use this if it fails.

Reply:
each time, the bundled js gets a unique name [here](https://github.com/huggingface/doc-build/tree/main/transformers/master/en/_app) as `start-{ID}.js`

unlike `rm -rf`, with `cp -r`, they will pile up (although they would not cause any rendering issue; will just look noisy)

Reply:
Will adjust! Good point!